File: ESAE_CROSSPLATFORM/ESAE_CROSSPLATFORM/ESAE_CROSSPLATFORM/Models/Propina.cs
using System;
using System.Collections.Generic;

namespace ESAE_CROSSPLATFORM.Models
{
    public class Propinas
    {
		public int id { get; set; }
		public string mes { get; set; }
		public int ano { get; set; }
        public int valor { get; set; }
        public bool pago { get; set; }
        public List<Curso> cursos { get; set; }
    }
}

File: finalbackoffice/backoffice/app/Propina.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use App\Curso;

class Propina extends Model
{
	protected $table = "propinas";
	protected $primaryKey = "id";
	protected $fillable = array("ano", "mes", "valor", "pago");
	public $timestamps = true;
	
	public function cursos()
	{
		return $this->belongsToMany('App\Curso', 'curso_propina')->withTimestamps();
	}
}
